pandas/core/arrays/boolean.py

```diff
@@ -273,7 +273,7 @@ def dtype(self) -> BooleanDtype:
 
     @classmethod
     def _from_sequence(
-        cls, scalars, *, dtype=None, copy: bool = False
+        cls, scalars, *, dtype: Optional[DtypeArg] = None, copy: bool = False
```

Comment:
most of these are Optional[Dtype} NOT DtypeArg (eg. only a single dtype and not a dict of them). The only things that take DtypeArg are things like astype and the I/O methods

Reply:
Changed it. I did it right last time, but I think I was a bit too quick with submitting this PR. Thanks for the review.